2023-08-11 16:48:41,608 INFO    MainThread:68588 [internal.py:wandb_internal():90] W&B internal server running at pid: 68588, started at: 2023-08-11 16:48:41.608090
2023-08-11 16:48:41,609 DEBUG   HandlerThread:68588 [handler.py:handle_request():141] handle_request: run_start
2023-08-11 16:48:41,609 INFO    WriterThread:68588 [datastore.py:open_for_write():75] open: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230811_164841-3kdapc5m/run-3kdapc5m.wandb
2023-08-11 16:48:41,611 DEBUG   HandlerThread:68588 [meta.py:__init__():35] meta init
2023-08-11 16:48:41,611 DEBUG   HandlerThread:68588 [meta.py:__init__():49] meta init done
2023-08-11 16:48:41,611 DEBUG   HandlerThread:68588 [meta.py:probe():209] probe
2023-08-11 16:48:41,612 DEBUG   HandlerThread:68588 [meta.py:_setup_git():199] setup git
2023-08-11 16:48:41,619 DEBUG   HandlerThread:68588 [meta.py:_setup_git():206] setup git done
2023-08-11 16:48:41,620 DEBUG   HandlerThread:68588 [meta.py:_save_pip():53] save pip
2023-08-11 16:48:41,620 DEBUG   HandlerThread:68588 [meta.py:_save_pip():67] save pip done
2023-08-11 16:48:41,620 DEBUG   HandlerThread:68588 [meta.py:probe():247] probe done
2023-08-11 16:48:56,734 WARNING MainThread:68588 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-08-11 16:48:57,142 WARNING MainThread:68588 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-08-11 16:48:57,142 ERROR   MainThread:68588 [internal.py:wandb_internal():156] Internal process interrupted.
2023-08-11 16:48:57,613 INFO    SenderThread:68588 [sender.py:finish():1111] shutting down sender
2023-08-11 16:48:57,744 INFO    WriterThread:68588 [datastore.py:close():279] close: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230811_164841-3kdapc5m/run-3kdapc5m.wandb
2023-08-11 16:48:57,744 INFO    HandlerThread:68588 [handler.py:finish():806] shutting down handler
2023-08-11 16:48:57,745 INFO    MainThread:68588 [internal.py:handle_exit():80] Internal process exited
